watson-developer-cloud/discovery-components
# fix: public storybook build

#### What do these changes do/fix?
Adds `CIDocument` to public storybook build


#### How do you test/verify these changes?
run `storybook:build:release` under `packages/discovery-component-react` and run the `index.html` file generated under `docs/storybook` you should see `CIDocument` story. 

#### Have you documented your changes (if necessary)?
Yes
#### Are there any breaking changes included in this pull request?
No
